### DISCUSSION TITLE
Showing some appreciation

### DISCUSSION THREAD

**Discussion by flukejones** [paperwm/PaperWM#795] [General]
There's no easy way on github to just say "Thanks for the work you've done" with a personal touch. So this "issue" is what I'll use.

Honestly, thanks for this extension, it's made a world of difference to me in just day-to-day desktop use and workflow. Incredible really.

(also, feel free to close or leave open)

**Comment by horshacktest** [comment]
I concur with your feelings. Come hang out on the PaperWM zulip space. I've been considering starting a “what I love most...” thread. 
https://paperwm.zulipchat.com/

**Comment by flukejones** [comment]
I tried Zulip, the UX doesn't seem very intuitive or friendly? Would a discord or rocketchat server be better?

**Comment by hedning** [comment]
Thanks, I'm glad it you are finding it useful :)

Hearing how people are using paperwm, what they like and what they dislike is always useful :+1: 

Regarding zulip, there's not that much activitiy going on so the UX might be a bit overkill, but we'll probably keep it for now tbh.

**Comment by ghost** [comment]
Couldn't agree more!

**Comment by jsonMartin** [comment]
Seriously. This extension is Amazing!!! I used to use other tiling window managers in the past, but I like the paradigm of PaperWM even more than tiling WMs such as i3 or DWM; I find the workflow works well to keep me organized. 

I also needed a new machine and previously would have gone OSX with the Amethyst window manager extension, but since I ended up discovering PaperWM in time and I've been having such a good experience using it on Manjaro, I decided to forgo buying a new Mac laptop and switch to this instead, saving a bunch of cash on hardware and making me more productive in the process. Thank you!!
